doc/api/v8.md

```diff
@@ -22,6 +22,7 @@ Returns an object with the following properties:
 * `total_available_size` {number}
 * `used_heap_size` {number}
 * `heap_size_limit` {number}
+* `does_zap_garbage` {bool}
```

Comment:
I think this should be `{number}` instead, if we're not actually returning a boolean type (`true` or `false`).